lib/json_web_token.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class JsonWebToken
+  class << self
+    def encode(payload, exp = 5.minutes.from_now)
+      payload[:exp] = exp.to_i
+      JWT.encode(payload, jwt_secret)
```

Comment:
Might want to be explicit about the hashing algorithm that is being used to encode the JWT

Reply:
Any suggestions for which one to use? Not sure what the differences are so I just used the default one